website/src/Components/Edit/edit.tsx

```diff
@@ -75,6 +75,17 @@ const Edit: React.FC<{
       is_contact_phone_number_correct.validate
     );
   };
+  //This is to get the contact info while keeping support for the legacy contact phone
+  function getContactInfo(): string {
+    if(currentData) {
+      if(currentData.contactInfo) {
+        return currentData.contactInfo
+      } else if(currentData.contactPhone) {
+        return currentData.contactPhone
+      }
+    }
+    return ""
+  }
```

Comment:
I wouldn't change the actual database entry because you end with cascading back-compat problems with type definitions in multiple places. 

Since ```contactPhone``` is equivalently types to ```contactInfo``` as a string, I would just add a comment to maybe our documentation instead that ```contactPhone``` can be any contact info. 

As in, the only thing that needs to change is just the UI and not the actual underlying data structure.

Reply:
Fixed!